### ISSUE TITLE
Cannot read property '1' of null

### ISSUE THREAD

**Issue by vanhumbeecka** [andrewvasilchuk/vue-lazy-youtube-video#4]
Happens when I pass the following YouTube URL: https://www.youtube-nocookie.com/embed/MTk-Hwr15ao.

Does not happen when using a *normal* YouTube URL, like: https://www.youtube.com/watch?v=2u_pZ-SgACk.

Not sure if this qualifies as a bug or a feature.

**Comment by unr** [comment]
This is caused by 

```js
		id() {
			const regExp = /^https:\/\/www\.youtube\.com\/watch\?v=(.+)$/;
			return regExp.exec(this.url)[1];
		},
```

Should be updated like so.

```js
		id() {
			try {
				const regExp = /^https:\/\/www\.youtube\.com\/watch\?v=(.+)$/;
				return regExp.exec(this.url)[1];
			} catch (err) {
				throw 'Youtube url in wrong format. Must be written as https://wwww.youtube.com/watch?v=1234';
				return 0;
			}
		},
```

**Comment by andrewvasilchuk** [comment]
@unr, @vanhumbeecka Thanks. I will change `regExp` a bit to  handle cases with url that starts with: 
`https://www.youtube-nocookie.com `. Also in the next minor version the `noCookies` prop will be introduced, which will generate appropriate url for the `<iframe />.

**Comment by andrewvasilchuk** [comment]
See 29a6ad58163af163c45ed7f20f461e2028d96b1d.

**Comment by andrewvasilchuk** [comment]
*Notice*: you should no pass url with `/embed/` part. Just pass a direct link to the video like this:
`https://www.youtube.com/watch?v=2u_pZ-SgACk`.

**Comment by andrewvasilchuk** [comment]
@vanhumbeecka, @unr, Fixed in `@1.3.0-beta.0`. Just pass `noCookie` prop with `true` value.

**Comment by andrewvasilchuk** [comment]
Can we close the issue? 🤔
